tasks/main.yml

```diff
@@ -113,6 +110,13 @@
   notify: verify config and restart httpd
   when: _use_tls
 
+- name: remove obsolete files
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+    - "{{ _vhost_confdir }}/python.conf"
+
```

Comment:
Well, I guess we can say we depreciate that in 6 months, 1 year ? I just like to have a expiration date to stuff we deprecate.

Realistically, all our servers will have got a run by then.